samcli/lib/generated_sample_events/event-mapping.json

```diff
@@ -917,6 +917,19 @@
     }
   },
   "s3": {
+    "batch-invocation": {
+      "filename": "S3BatchInvocation",
+      "help": "Generates an Amazon S3 Batch Operations Invocation Event",
+      "tags": {
+        "bucket-arn": {
+          "default": "arn:aws:s3:us-east-1:0123456788:examplebucket"
```

Comment:
s3 bucket arns are global, more information here: https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-arn-format.html

Would you be able to adjust based on the doc?

Thanks for the PR!

Reply:
updated with proper bucket syntax, thanks!